bigcache.go

```diff
@@ -107,6 +107,19 @@ func (c *BigCache) Len() int {
 	return len
 }
 
+// Stats returns cache's statistics
+func (c *BigCache) Stats() Stats {
```

Comment:
Can you add a test for this method? Best if you can extend current tests with additional assertion